instrumented/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,254 @@
+                      id="email"
+                      required="required"
+                      aria-required="true"
+                      className="input"
+                      onChange={(event) => this.handleInputChange(event)}/>
+                  </div>
+                  <label className="preferred-lang-p label" htmlFor="language">Preferred language<em>*</em></label>
+                  <div className="label-input-container">
+                    <input type="radio"
+                      name="language"
+                      id="english"
+                      value="en"
+                      required="required"
+                      className="input-radio"
+                      checked={this.state.language === "en"}
+                      onChange={(event) => this.handleInputChange(event)}/>
+                    <label className="label-radio" htmlFor="en">English</label>
+                  </div>
+                  <div className="label-input-container">
+                    <input type="radio"
+                      name="language"
+                      id="spanish"
+                      value="es"
+                      required="required"
+                      className="input-radio"
+                      checked={this.state.language === "es"}
+                      onChange={(event) => this.handleInputChange(event)}/>
+                    <label className="label-radio" htmlFor="es">Spanish</label>
+                  </div>
+                  <div className="label-input-container">
+                    <input type="checkbox"
+                      name="agree_to_emails"
+                      id="agree_to_emails"
+                      value={this.state.agree_to_emails}
+                      required="required"
+                      aria-required="true"
+                      className="input-checkbox"
+                      checked={this.state.agree_to_emails}
+                      onChange={() => this.handleCheckboxInput()}/>
+                    <label className="agree-to-emails-checkbox label" htmlFor="agree_to_emails">
+                      Sign up for email notifications about upcoming elections in my state.
+                    </label>
+                  </div>
+                  <div className="submit-button-container">
+                    <button
+                      className="submit-button"
+                      onClick={(event) => this.handleSubmit(event)}>
+                        Submit
+                    </button>
+                  </div>
+                  <div className="missing-input-message-container">
+                    {this.state.displayMissingInput ? this.displayMissingInputMessage() : null}
+                    <p className="server-message">{this.state.serverMessage}</p>
+                  </div>
+                </div>
+              </form>
+            }
+          </>
+        }
+      </>
+    );
+  }
+}
+
+export default EmailNotificationForm;
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
Instrumented files are needed for coverage tests.  Essentially, instrumenting adds numbering to the code so when the cypress-coverage package is ran, it can evaluate whether or not a line was run by the tests.  At least that's my basic understanding of it!